implementations/rust/ockam/ockam_node/src/delayed.rs

```diff
@@ -110,166 +110,122 @@ mod tests {
     }
 
     #[allow(non_snake_case)]
-    #[test]
-    fn scheduled_3_times__counting_worker__messages_count_matches() -> Result<()> {
-        let (mut ctx, mut executor) = NodeBuilder::without_access_control().build();
-        executor
-            .execute(async move {
-                let msgs_count = Arc::new(AtomicI8::new(0));
-                let mut heartbeat =
-                    DelayedEvent::create(&ctx, "counting_worker", "Hello".to_string())
-                        .await
-                        .unwrap();
-
-                let worker = CountingWorker {
-                    msgs_count: msgs_count.clone(),
-                };
-
-                ctx.start_worker("counting_worker", worker).await.unwrap();
-
-                heartbeat
-                    .schedule(Duration::from_millis(100))
-                    .await
-                    .unwrap();
-                sleep(Duration::from_millis(150)).await;
-                heartbeat
-                    .schedule(Duration::from_millis(100))
-                    .await
-                    .unwrap();
-                sleep(Duration::from_millis(150)).await;
-                heartbeat
-                    .schedule(Duration::from_millis(100))
-                    .await
-                    .unwrap();
-                sleep(Duration::from_millis(150)).await;
-
-                assert_eq!(3, msgs_count.load(Ordering::Relaxed));
-
-                ctx.stop().await.unwrap();
-            })
-            .unwrap();
+    #[ockam_macros::test(crate = "crate")]
+    async fn scheduled_3_times__counting_worker__messages_count_matches(ctx: &mut Context) -> Result<()> {
+        let msgs_count = Arc::new(AtomicI8::new(0));
+        let mut heartbeat =
+            DelayedEvent::create(&ctx, "counting_worker", "Hello".to_string())
+                .await?;
+
+        let worker = CountingWorker {
+            msgs_count: msgs_count.clone(),
+        };
 
+        ctx.start_worker("counting_worker", worker).await?;
+
+        heartbeat
+            .schedule(Duration::from_millis(100))
+            .await?;
+        sleep(Duration::from_millis(150)).await;
+        heartbeat
+            .schedule(Duration::from_millis(100))
+            .await?;
+        sleep(Duration::from_millis(150)).await;
+        heartbeat
+            .schedule(Duration::from_millis(100))
+            .await?;
+        sleep(Duration::from_millis(150)).await;
+
+        assert_eq!(3, msgs_count.load(Ordering::Relaxed));
```

Comment:
I think , `ctx.stop().await.unwrap();` is missing from rewritten test could you please fix that?

Reply:
Ok, I'll check all mentioned,  now traveling 👍